src/passkit.cs

```diff
@@ -1036,6 +1078,18 @@ interface PKPaymentAuthorizationControllerDelegate {
 		[Watch (4,0)][iOS (11,0)]
 		[Export ("paymentAuthorizationController:didSelectPaymentMethod:handler:")]
 		void DidSelectPaymentMethod (PKPaymentAuthorizationController controller, PKPaymentMethod paymentMethod, Action<PKPaymentRequestPaymentMethodUpdate> completion);
+
+		[Watch (7,0)][iOS (14,0)]
+		[Export ("paymentAuthorizationController:didRequestMerchantSessionUpdate:")]
+		void DidRequestMerchantSessionUpdate (PKPaymentAuthorizationController controller, Action<PKPaymentRequestMerchantSessionUpdate> handler);
+	
+		[Watch (7,0)][iOS (14,0)]
+		[Export ("presentationWindowForPaymentAuthorizationController:")]
+		[return: NullAllowed]
+#if MONOMAC
```

Comment:
I'm assuming this should eventually be abstract for all platforms, so:

```suggestion
#if MONOMAC || XAMCORE_4_0
```

Reply:
no, it's @required on macOS and (still?) @optional on iOS / watchOS
probably because they don't want to introduce a breaking change (can't blame them to do the right thing, even if they don't generally care)